ssa.go

```diff
@@ -1253,16 +1262,23 @@ func (s Subtitles) WriteToSSA(o io.Writer) (err error) {
 		var b = []byte("\n[Events]\n")
 
 		// Format
-		var formatMap = make(map[string]bool)
 		var format = []string{
```

Comment:
Also could you add the following comment above `format`?

```
We need to declare those 9 columns here otherwise VLC doesn't display subtitles properly column
```

Reply:
sure, i've added it on the last  commit.

Reply:
I made a typo 😱 Could you remove the `column` at the end of the comment?